docs/source/intro.rst

```diff
@@ -0,0 +1,61 @@
+************
+Introduction
+************
+AMD **ROCm** has two classification of libraries,
+
+- **roc**\* : AMD GPU Libraries, written in `HIP <https://rocmdocs.amd.com/en/latest/Programming_Guides/HIP-GUIDE.html>`_.
+- **hip**\* : A Thin interface between the AMD **roc*** and Nvidia **cu*** library.
+
```

Comment:
I think the following is better "AMD CPU library that is a thin interface to either AMD **roc** or Nvidia **cu** libraries."